docs/topics/auth/default.txt

```diff
@@ -122,20 +122,19 @@ Authenticating users
     form of keyword arguments, for the default configuration this is
     ``username`` and ``password``, and it returns
     a :class:`~django.contrib.auth.models.User` object if the password is valid
-    for the given username. If the password is invalid,
+    for the given username **and** if the ``user.is_active==True``. 
```

Comment:
The bit about `is_active==True` is a feature of the default authentication backend, not `authenticate()` itself. I'd rather explain that authenticate uses the authentication backends rather than describe a behavior of the backend.